src/CoreShop/Bundle/StoreBundle/DependencyInjection/Configuration.php

```diff
@@ -107,9 +107,11 @@ private function addPimcoreResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('store_item')->defaultValue('/bundles/coreshopstore/pimcore/css/store.css')->end()
                         ->end()
                     ->end()
-                    ->scalarNode('permissions')
-                        ->cannotBeOverwritten()
-                        ->defaultValue(['store'])
+                    ->arrayNode('permissions')
```

Comment:
don't overwrite permission configurations, you cannot change them anyway and its not part of this PR